app/views/v1/shared/_post.json.jbuilder

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```

Comment:
I think you can simplify this into 
```
json.extract! post, :id, : content, : created_at, : updated_at
json.author_url "/v1/users/#{post.author.username}"
json.posted_to post.postable.username
json.postable_url "/v1/users/#{post.postable.username}"
json.content post.content
```
unless you want in specific order try playing and look at the examples on https://github.com/rails/jbuilder

Reply:
Thanks @roseliux! I will see how I can incorporate this suggested change. It's not my first time to use jbuilder but I haven't really gone through all the tools it provides. 



> IMHO all looks really clean, congrats, really like that you implement rubocop and unit test 1st_place_medal

Thanks for taking a look at it. So do you think it's good enough to merge to the base branch?

Reply:
yes I will approve 👍

Reply:
> I think you can simplify this into
> 
> ```
> json.extract! post, :id, : content, : created_at, : updated_at
> json.author_url "/v1/users/#{post.author.username}"
> json.posted_to post.postable.username
> json.postable_url "/v1/users/#{post.postable.username}"
> json.content post.content
> ```
> 
> unless you want in specific order try playing and look at the examples on https://github.com/rails/jbuilder

Change made! :)